app/controllers/concerns/resource_controller.rb

```diff
@@ -15,6 +15,15 @@ def new
       redirect_to new_scanned_resource_path
     else
       @change_set = change_set_class.new(new_resource, append_id: params[:parent_id]).prepopulate!
+      # If there's a parent, authorize updating it instead.
```

Comment:
Maybe move this comment to the method itself, adding a tiny bit more about why. I think elsewhere in this PR the logic exists that if the parent can be edited then the folder can be created, but it's not clear here that the update authorization also confers the create authorization.

Reply:
Oh, yeah. This is just left behind from before a refactor. 😅